packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -233,12 +238,31 @@ export class Pane extends Foundation<PaneHandledProps, PaneUnhandledProps, PaneS
         return (
             <button
                 className={this.props.managedClasses.pane_resizeHandle}
-                aria-hidden={true}
                 onMouseDown={this.onMouseDown}
+                onKeyDown={this.onKeyDown}
+                aria-hidden={true}
             />
         );
     }
 
+    /**
+     * Handle keyPress
+     */
+    public onKeyDown = (e: React.KeyboardEvent<HTMLButtonElement>): void => {
+        const isShift: boolean = e.shiftKey;
+
+        switch (e.keyCode) {
+            case KeyCodes.arrowLeft:
+                this.setWidth(this.rootElement.current.clientWidth - (isShift ? 10 : 1));
```

Comment:
Assuming this math works in RTL?

Reply:
I believe it should, might want to validate though.

Reply:
I’ve manually tested for quite a bit - we do need to add tests to this package all up though.

Reply:
Since we’re using Right/Left keys to increment I think we’ll need to modify based on RTL - thanks @bekell for catching that!

Reply:
@bekell I’m seeing a larger issue here with RTL support; I’m going to take a separate issue and address there as it affects the click behavior as well. I’d rather align/fix both events in a separate, unified PR.